src/Captioning/Format/JsonFile.php

```diff
@@ -20,16 +20,14 @@ public function parse()
             throw new \InvalidArgumentException('Invalid JSON subtitle');
         }
 
-        if (count($decodedContent['captions']) == 0) {
-            return false;
-        }
-
-        foreach ($decodedContent['captions'] as $c) {
-            $cue = new JsonCue($c['startTime'], $c['startTime'] + $c['duration'], $c['content']);
-            $cue->setDuration($c['duration']);
-            $cue->setStartOfParagraph($c['startOfParagraph']);
-
-            $this->addCue($cue);
+        if (count($decodedContent['captions']) > 0) {
+            foreach ($decodedContent['captions'] as $c) {
+                $cue = new JsonCue($c['startTime'], $c['startTime'] + $c['duration'], $c['content']);
+                $cue->setDuration($c['duration']);
+                $cue->setStartOfParagraph($c['startOfParagraph']);
+
+                $this->addCue($cue);
+            }
```

Comment:
I don't think so. Also it matches interface, because method should return `$this` instead of bool.